<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuEdgeContraction</class>
 <widget class="QDialog" name="MenuEdgeContraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edge contraction</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2">
    <widget class="QSpinBox" name="spinBox_nb_remaining_vertices">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QCheckBox" name="checkBox_link_condition">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Link condition</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="txt_nb_collapses">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Policy</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="m_simplificationMethod">
     <item>
      <property name="text">
       <string>Length_midpoint</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" alignment="Qt::AlignRight">
    <widget class="QLabel" name="txt_nb_vertices">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Number of vertices</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Number of contractions</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Number of vertices after </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButton_collapse">
     <property name="text">
      <string>Perform collapse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
